qa/tasks/vstart_runner.py

```diff
@@ -462,8 +470,8 @@ def _get_pid(self):
         Return PID as an integer or None if not found
         """
         ps_txt = six.ensure_str(self.controller.run(
-            args=["ps", "ww", "-u"+str(os.getuid())]
-        ).stdout.getvalue()).strip()
+            args=["ps", "ww", "-u"+str(os.getuid())], stdout=StringIO()).\
+            stdout.getvalue()).strip()
```

Comment:
Maybe the `six.ensure_str` makes no sense any more and at the same time we will only suppor the py3.

Reply:
Right.

Reply:
Removed this one as well as few other identical instances in this patch.